src/main/java/com/github/cameltooling/lsp/internal/instancemodel/propertiesfile/CamelPropertyEntryInstance.java

```diff
@@ -57,11 +58,11 @@ public CamelPropertyEntryInstance(String line, Position startPosition, TextDocum
 		camelPropertyValueInstance = new CamelPropertyValueInstance(camelPropertyFileValueInstanceString, camelPropertyKeyInstance, textDocumentItem);
 	}
 	
-	public CompletableFuture<List<CompletionItem>> getCompletions(Position position, CompletableFuture<CamelCatalog> camelCatalog) {
+	public CompletableFuture<List<CompletionItem>> getCompletions(Position position, CompletableFuture<CamelCatalog> camelCatalog, CamelKafkaConnectorCatalogManager camelKafkaConnectorManager) {
 		if (position.getCharacter() <= camelPropertyKeyInstance.getEndposition()) {
```

Comment:
Aurelien, I am wondering if it might be good for code readability to extract that check into a function like `isPropertyKeyCompletion(...)`. Its really a minor thing but would make it easier to read. What do you think?